static/src/javascripts/projects/common/views/experiments/quick-survey.html

```diff
@@ -0,0 +1,92 @@
+<style>
+
+.impressions-survey__heading {
+    margin-top:0;
+    padding: 0.5em 1em 0.5em 1em;
+    font-size:0.8em;
+    background-color:#AAD7E4;
+}
+
+.impressions-survey__title {
+    display: inline-block;
+    font-family: "Guardian Text Sans Web", "Helvetica Neue", "Helvetica", "Arial", "Lucida Grande", "sans-serif";
+    font-weight:bold;
+    font-size: 1.8em;
+    margin-right: 1em;
+}
+
+.impressions-survey__about {
+    font-weight:normal;
+    color: inherit;
+    text-decoration: none;
+    padding-left: 1em;
+    border-left: solid #fff;
+}
+
+.impressions-survey__about:hover,
+.impressions-survey__about:link,
+.impressions-survey__about:active,
+.impressions-survey__about:visited {
+    text-decoration: none;
+}
+
+.impressions-survey__body {
+    color:#fff;
+    margin-bottom: 1em;
+    padding: 1em 1em 1em 3em;
+    font-size: 0.8em;
+    background-color: #005689;
+    opacity: 1;
+    position:relative;
+}
+
+.impressions-survey__question {
+    font-weight:bold;
+    margin-left:-1em!important;
+}
+
+.fi-survey__button, .fi-survey__label {
+    margin: 0 0.2em 0 0.2em;
+}
+
+.impressions-survey__thanks {
+    color:#fff;
+    font-size:2em;
+    font-weight:bold;
+    position:absolute;
+    top:20%;opacity:0;
+    visibility:collapse;
+}
+
+</style>
+
+<div class="impressions-survey__heading">
+    <div class="impressions-survey__title">Survey</div>
+    <div class="impressions-survey__title"><a href="https://www.theguardian.com/help/terms-of-service" target="_blank" class="impressions-survey__about">About</a></div>
+</div>
+
+
+
+<div class="impressions-survey__body" data-link-name="impressions-frequency-survey">
+
+    <div id="surveyTextbox">
+        <h3 class="impressions-survey__question">
+            How often do you read the Guardian in a digital format?</h3>
+        <div id="impressions-survey__select">
+            <label class="fi-survey__label"><input type="radio" class="fi-survey__button" data-link-name="frequency_5" value="frequency_5">
+                Every day/most days</label><br>
```

Comment:
I would not use a `<br>` for formatting the labels. You could just set the display type on `fi-survey__label` to be `block` ie `.fi-survey__label { display:block }`